src/components/organization/organization.service.ts

```diff
@@ -88,54 +86,15 @@ export class OrganizationService {
   }
 
   async readOne(orgId: string, session: ISession): Promise<Organization> {
-    const result = await this.db
```

Comment:
You can't remove this yet, there are services that have not migrated to the security group model and are still looking for ACL.